cmd/storj-sim/network.go

```diff
@@ -269,6 +269,11 @@ func newNetwork(flags *Flags) (*Processes, error) {
 			)
 		}
 
+		if flags.Redis != "" {
+			process.Arguments["setup"] = append(process.Arguments["setup"],
+				"--server.revocation-dburl", flags.Redis)
```

Comment:
One important thing here is that we can have multiple satellites and they should not share the same database.

Reply:
oh good point

Reply:
We might want them to share a revocation db though.

Reply:
Not really, the purpose here is more to ensure that things work with separate satellites that have no knowledge of each other.

Reply:
My point is that it would be beneficial to share knowledge of revisions across satellites.

Reply:
How do we do that when we have ten partners each with their own revocation database implementation running independently none of which have direct access to each other?

Reply:
I'm only taking about stuff we control. Obviously we can't do that in your example.

Reply:
Yeah, I agree. The purpose here simulating multiple satellites is to simulate independent satellites. Sure we could also try to simulate cooperating satellites, but that in my mind would be secondary.

Reply:
i'll see if i can spin up a unique miniredis server for each satellite